### PR TITLE
Fix typos in new GUI docs

### PR DESCRIPTION
**Pull request by morganwillcock**

### CODE REVIEW

File: sphinx/source/gui.rst [hunk 1]
```diff
@@ -363,15 +363,15 @@ documented above, the following variables exist:
...
-    The color used by focused items in the gui, including the text of
+    The color used by focused items in the GUI, including the text of
     of buttons and the thumbs (movable areas) of sliders and scrollbars.
 
 .. var:: gui.selected_color = '#555555'
 
     The color used by the text of selected buttons. (This takes priority
     over the hover and idle colors.)
 
-.. var:: gui.insensitive_color = '#8888887f'
```
Comment (xavi-mat) [inline sphinx/source/gui.rst]: Is this a typo? I thought this is gray with some level of transparency (~50%)

Comment (morganwillcock) [inline sphinx/source/gui.rst]: Ah, could be. But I just followed the description in the same document:

`These variables should generally be set to hexadecimal color codes, which are strings of the form "#rrggbb"`

**Comment by morganwillcock** [comment]
I've updated it, assuming that the alpha component is valid for use everywhere.

**Comment by renpytom** [comment]
Merged, thank you.

**Pull request merged**